docs/packaging/packaging-practices.md

```diff
@@ -286,7 +277,7 @@ security/cve-xxxx-xxxx.patch
 fix-silliness.patch
 ```
 
-Both of the files above will be applied using `-p1`. If you need to use stripping num, like `-p4`, you can do something like:
+Both of the files above will be applied using `-p1`. If you need to use a different stripping num, like `-p4`, you can do something like:
```

Comment:
```suggestion
Both of the files above will be applied using stripping number `-p1`. If you need to use a different stripping number, like `-p4`, you can write:
```

Reply:
Yeah, I wasn't sure about this one, but kept it close to the old version in the end. But this is more readable for sure. Done.